core/src/test/java/org/kestra/core/tasks/BashTest.java

```diff
@@ -149,4 +150,26 @@ void longBashCreateTempFiles() throws Exception {
         assertThat(run.getStdOut().size(), is(2));
         assertThat(run.getStdErr().size() > 0, is(false));
     }
+
+    @Test
+    void useInputFiles() throws Exception {
+        RunContext runContext = runContextFactory.of();
+
+        HashMap<String, String> files = new HashMap<String, String>();
```

Comment:
```suggestion
        Map<String, String> files = new HashMap<String, String>();
```
Keep interface always on left please